src/chrome/content/rules/Flickr.xml

```diff
@@ -46,7 +46,8 @@
 	<!--securecookie host="^\.flickr\.com$" name="^(BX|admsort|cookie_session|current_identity|density|ffs|fldetectedlang|flrb|flrbcr|flrbgdrp|flrbgmrp|flrbgrp|flrbp|flrbs|fltoto|just_registered|localization|memsort|modsort|photoPHS|photoPWS|vp|xb)$" /-->
 	<!--securecookie host="^www\.flickr\.com$" name="^(liqph|liqpw|ywandp)" /-->
 
-	<securecookie host=".*\.flickr\.com$" name="^(?!current_identity$).+" />
+	<!-- If we secure this cookie, login fails. -->
+	<!-- <securecookie host=".*\.flickr\.com$" name="^(?!current_identity$).+" /> -->
```

Comment:
This one in particular I chose to comment rather than deleting, because it's been fixed in the past and has reappeared. So it's useful to have a comment saying "don't do this."